### PR TITLE
refactor: make dev users non-chain-specific

### PR DESCRIPTION
**Pull request by harrysolovay**
This change exposes the `createUsers` util through `capi` instead of through chain-specific codegen (the dev users are not chain-specific, after all). Also, given that we use the term "dev" in the context of dev nets, let's do the same for test users. The creation util is now exposed through `capi` itself, so let's rename `createUsers` to `createDevUsers`; hopefully this makes clear that its not to be used outside of the context of Capi devnets.

### CODE REVIEW

File: devnets/devnets_env.ts
Comment (tjjfvi) [inline devnets/devnets_env.ts:1]: I think this should be renamed to devnetsUrl.ts

File: examples/dev/test_users.eg.ts
Comment (tjjfvi) [inline examples/dev/test_users.eg.ts:1]: Should be renamed

File: server/codegenHandler.ts [hunk 1]
```diff
@@ -212,30 +212,16 @@ export function createCodegenHandler(dataCache: CacheBase, tempCache: CacheBase)
...
-import * as C from "./capi.js"
+    `import * as C from "./capi.js"
 
 export const connect = C.${connection.type}.bind(${JSON.stringify(connection.discovery)})
-
-${
-      connection.type === "DevnetConnection"
-        ? `export const createUsers = C.testUserFactory(${JSON.stringify(connection.discovery)})`
-        : ""
-    }
```
Comment (tjjfvi) [inline server/codegenHandler.ts:224]: Yay

**Review by kratico — DISMISSED** [review]
LGTM

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**